console/services/app_config/port_service.py

```diff
@@ -282,6 +284,9 @@ def manage_port(self, tenant, service, region_name, container_port, action, prot
             code, msg = self.__change_protocol(tenant, service, deal_port, protocol)
         elif action == "change_port_alias":
```

Comment:
`change_port_alias` `update_k8s_service_name` should merge?

Reply:
fixed

Reply:
why remove change_port_alias？

Reply:
understood. It's ok this time.